### ISSUE TITLE
Misaligned address bar

### ISSUE THREAD

**Issue by acdavit** [soup-bowl/Modoki-Firefox#32]
### What version of the theme are you using?

Latest Version (0.7)

### What edition of Firefox are you on?

Mainline Firefox

### Please describe the problem you are experiencing. Screenshots very welcome.

The address bar is off center and misaligned. 

Firefox v130.0 on Arch Linux

![Screenshot_2024-09-05_23-42-33](https://github.com/user-attachments/assets/f88f7d34-f401-4870-bb0a-51401872f0f7)

**Comment by soup-bowl** [comment]
Yes this has been bothering me immensely to the point I've personally stopped using the theme. It's on my radar to fix, but the whole bar needs refactoring (and looks like in new versions they've changed something that invalidates it entirely) so it's on my radar for when I've got time.
